woocommerce/woocommerce-ios
# System Status Report: Add UI for system status report screen

Part of #5071 


### Description

This PR add new screen for system status report.
The screen uses a view model to fetch system status from remote. Error handling has been added, but the parsing for content will be updated in a separate PR.

### Testing instructions

- Log in to a valid account
- On My Store tab, navigate to Settings screen.
- Select Help & Support > System Status Report.
- Notice that you'll be navigated to System Status Report screen with a loading indicator indicating that the status is being fetched. If an error occurs during this step, an alert will be displayed with option to retry fetching.

### Screenshots


<img src="https://user-images.githubusercontent.com/5533851/146338101-95be2cf2-d3fd-470d-a9c2-b683ed0a3e74.png" width=320 />
<img src="https://user-images.githubusercontent.com/5533851/146338113-aaf7733b-38f5-4535-9a61-17c82246c66b.png" width=320 />

---
